daphne/src/roles/mod.rs

```diff
@@ -1222,27 +1228,21 @@ mod test {
             );
         }
 
-        // Get AggregationJobResp and then extract the transition data from inside.
-        let agg_job_resp = AggregationJobResp::get_decoded(
-            &helper::handle_agg_job_req(&*t.helper, &req)
-                .await
-                .unwrap()
-                .payload,
-        )
-        .unwrap();
-        let transition = &agg_job_resp.transitions[0];
+        let query = task_config.query_for_current_batch_window(t.now);
+        let req = t.gen_test_coll_job_req(query, task_id).await;
+        leader::handle_coll_job_req(&*t.leader, &req).await.unwrap();
```

Comment:
We need to change this test because, in draft02, the report now gets rejected on the second request for the job rather than the first. This behavior is expected.